extension/zooarchnet/chronometricDate.xml

```diff
@@ -11,22 +11,22 @@
-    <property name='chronometricDateProtocol' namespace='http://zooarchnet.org/dwc/terms/' qualName='http://zooarchnet.org/dwc/terms/chronometricDateProtocol' dc:description='A description or reference to the methods used to determine the chronometricDate.' examples='' type="string" required='false'/>
-    <property name='chronometricDateReferences' namespace='http://zooarchnet.org/dwc/terms/' qualName='http://zooarchnet.org/dwc/terms/chronometricDateReferences' dc:description='A list (concatenated and separated) of identifiers (publication, bibliographic reference, global unique identifier, URI) of literature associated with the chronometricDate.' examples='' type="string" required='false'/>
-    <property name='chronometricDateRemarks' namespace='http://zooarchnet.org/dwc/terms/' qualName='http://zooarchnet.org/dwc/terms/chronometricDateRemarks' dc:description='Notes or comments about the chronometricDate.' examples='' type="string" required='false'/>
+    <property name="chronometricDateID" namespace="http://zooarchnet.org/dwc/terms/" qualName="http://zooarchnet.org/dwc/terms/chronometricDateID" dc:description='An identifier for the set of information associated with a ChronometricDate. May be a global unique identifier or an identifier specific to the data set.' examples='GUID reference to the combination of information for one ChronometricDate.' type="string" required="true"/>
+    <property name='verbatimChronometricAge' namespace='http://zooarchnet.org/dwc/terms/' qualName='http://zooarchnet.org/dwc/terms/verbatimChronometricAge' dc:description='The verbatim age for a specimen, whether reported by a dating assay, associated references, or legacy information. For example, this could be the conventional radiocarbon age as given in an AMS dating report. This could also be simply what is reported as the age of a specimen in legacy collections data.' examples='27 BC to 14 AD' type="string" required='false'/>
+    <property name='verbatimChronometricAgeConversionProtocol' namespace='http://zooarchnet.org/dwc/terms/' qualName='http://zooarchnet.org/dwc/terms/verbatimChronometricAgeConversionProtocol' dc:description='The method used for converting the verbatimChronometricAge into a chronometric date in years, as captured in the maximumChronometricAge, maximumChronometricAgeReferenceSystem, minimumChronometricAge, and minimumChronometricAgeReferenceSystem fields. For example, calibration of conventional radiocarbon age or the currently accepted age range of a cultural or geological period.' examples='' type="string" required='false'/>
+    <property name='maximumChronometricAge' namespace='http://zooarchnet.org/dwc/terms/' qualName='http://zooarchnet.org/dwc/terms/maximumChronometricAge' dc:description='Upper limit for the age of a specimen as determined by a dating method.' examples='27' type="integer" required='false'/>
```

Comment:
This is the released extension that has several terms renamed which is not allowed